app/retail/templates/emails/gdpr_reconsent.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'emails/template.html' %}
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n humanize static %}
+
+{% block content %}
+
+<p>
+    Hello there,
+</p>
+<p>
+You may have heard about the new General Data Protection Regulation ("GDPR"), that comes into effect May 25, 2018. To help comply with GDPR consent requirements, we need to confirm that you would like to receive content from us.
+</p>
+<p>
+We hope that our content is useful to you. If you'd like to continue hearing from us, please update your subscription settings.
+</p>
+<p>
+<a class="button" href="https://gitcoin.co/settings/email/{{subscriber.priv}}">
```

Comment:
We could use reverse with request path here, so we don't have to update urls throughout the codebase if a path changes.